app/scripts/update-problem.js

```diff
@@ -0,0 +1,45 @@
+// module level exports
+const opn = require('opn');
+const {runApp} = require('../server/app');
+
+const {
+  CONTENT,
+  CONTENTRC,
+  CONTENTRCNAME,
+  PATH,
+  TEMPLATES,
+} = require('../config');
+
+const {
+  fsMkdir,
+  fsRead,
+  fsStat,
+  fsWrite,
+  fsWriteIfExists,
+  mkdirp,
+} = require('../utils');
+
+/*
+ *  @function updateProblem
+ */
+const updateProblem = (argv) => {
+  
+  const uuid = argv._[1];
+  const problemDirName = `${CONTENT}/${uuid}`;
+
+  // UUID Error Checks
+  if (!uuid) { console.log("Error: Please select a UUID."); return; } 
+  fsStat(problemDirName)
+    .then(_ => { return runApp() })
+    .then(_ => {
+        console.log(`Launching: http://localhost:3000/?uuid=${uuid}`);
+        opn(`http://localhost:3000/?uuid=${uuid}`);
```

Comment:
Worth making an issue for this - we probably want to create a `PORT` variable in config so that this can be controlled more easily. (Don't have to do this now, simply musing)